tools/travis-setup-db.sh

```diff
@@ -1,8 +1,11 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Is `env` is more likely to be in `/usr/bin` than `bash`?

Reply:
@kzemek we use `/usr/bin/env` in many scripts, it's just another way to do the same thing.
Never seen a system without` /bin/bash` or without `/usr/bin/env`.

Reply:
`/usr/bin/env <sth>` is a trick to find `<sth>` in caller's `$PATH`.
I'm not saying that the change is bad. I'm not saying it's good either.

Reply:
Feeling the same.
But there was some conversation on hipchat about using env everywhere. It was more about "one way to do this", and env was chosen.
Maybe we should document it.